src/main/kotlin/com/swm_standard/phote/dto/ExamDtos.kt

```diff
@@ -47,6 +47,26 @@ data class ReadExamResultsResponse(
     val students: List<ReadExamStudentResult>,
 )
 
+data class ReadExamResultDetail(
+    val statement: String,
+    val options: List<String>?,
+    val image: String?,
+    val category: Category,
+    val answer: String,
+    val submittedAnswer: String?,
+    val isCorrect: Boolean,
+    val sequence: Int,
+)
+
+data class ReadExamResultDetailResponse(
+    val examId: UUID,
+    val memberId: UUID,
```

Comment:
```suggestion
    val name: String,
```

여기 member id 대신 member 이름 담아주는거 어떨까영

Reply:
그게 낫겠다!! 바꿨시유